src/test/java/se/bjurr/gitchangelog/internal/settings/SchemaTest.java

```diff
@@ -0,0 +1,29 @@
+package se.bjurr.gitchangelog.internal.settings;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+import java.io.File;
+import java.net.MalformedURLException;
+
+@RunWith(Parameterized.class)
+public class SchemaTest {
+    
+    private File file;
+
+    public SchemaTest(File jsonSchemaFile) {
+        this.file = jsonSchemaFile;
+    }
+    
+    @Test
+    public void testSchema() throws MalformedURLException {
```

Comment:
Perhaps rename it to "testThatAllExamplesAreValid" if that os what id tests.

Reply:
done